### PR TITLE
[codewhisperer]: emit perceived latency telemetry

### PR DESCRIPTION
**Pull request by zixlin7**
## Problem
We need more visibility into user's perceived latency
## Solution
To emit telemetry on the new perceivedLatency event: the definition of this event is added in this [PR](https://github.com/aws/aws-toolkit-common/pull/366)
corresponding JB PR is [here](https://github.com/aws/aws-toolkit-jetbrains/pull/3298)
Screen Shot 2022-09-12 at 2 21 16 PM

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by zixlin7** [comment]
> Who is working on the JB side change?

I will raise a PR for JB shortly

File: src/codewhisperer/service/inlineCompletion.ts [hunk 1]
```diff
@@ -401,6 +405,21 @@ export class InlineCompletion {
                     this.setRange(new vscode.Range(editor.selection.active, editor.selection.active))
                     try {
                         await this.showRecommendation(editor)
+                        let languageId = editor?.document?.languageId
+                        languageId =
+                            languageId === CodeWhispererConstants.typescript
+                                ? CodeWhispererConstants.javascript
+                                : languageId
+                        const languageContext = runtimeLanguageContext.getLanguageContext(languageId)
```
Comment (leigaol) [inline src/codewhisperer/service/inlineCompletion.ts:413]: This will be conflicting with https://github.com/aws/aws-toolkit-vscode/pull/2859

Comment (Will-ShaoHua) [reply src/codewhisperer/service/inlineCompletion.ts:413]: this can go first, I will merge and resolve conflicts thereafter

Comment (zixlin7) [reply src/codewhisperer/service/inlineCompletion.ts:413]: thank you @Will-ShaoHua. I can rebase on yours since yours is already good to merge?

**Pull request merged**